magnolia/shared/src/main/scala-dotty/zio/config/magnolia/Constant.scala

```diff
@@ -0,0 +1,21 @@
+package zio.config.magnolia
+
+import zio.config._
+
+/**
+ * A copy of zio-config-derivation util, to be able to map a constant value
+ * to case class that has the same name as that of the constant value, with zero
+ * number of fields in it.
+ */
+case class Constant(value: String) extends PropertyType[String, String] {
```

Comment:
missing final

Reply:
Yes, absolutely it should be final.
These will be added as soon as the PR moves from Draft to an active one.